accounty-app/package.json

```diff
@@ -43,9 +43,13 @@
     "react-app-polyfill": "^0.1.3",
     "react-dev-utils": "^6.0.5",
     "react-dom": "^16.6.0",
+    "react-redux": "^5.1.0",
```

Comment:
very good!! you still did not implemented a store in out project.
implement a store and i will approve the PR